diff --git a/example_pkg.sv.orig b/example_pkg.sv
index 94eaa1f..63b87e0 100644
--- a/example_pkg.sv.orig
+++ b/example_pkg.sv
@@ -1,7 +1,5 @@
 
 // this line will stay
-// this is something which will get removed
-// another line to be removed
 
 // enterprise-grade code
 `define VALUE_0 0
@@ -10,10 +8,10 @@
 
 package example_pkg;
 
-  function bit [31:0] add_one(bit [31:0] a);
+  function logic [31:0] add_one(logic [31:0] a);
     // add one to the input value
-    bit [31:0] tmp;
-    tmp = a + `VALUE_1;
+    logic [31:0] tmp;
+    tmp = a + 1;
     return tmp;
   endfunction
 
@@ -21,4 +19,7 @@ package example_pkg;
     return b + `VALUE_2;
   endfunction
 
+  // adding a couple of lines here to verify the patcher
+  // more lines here
+
 endpackage
